dev-client/src/screens/ProjectInputScreen/SoilPitSettings.tsx

```diff
@@ -0,0 +1,136 @@
+import {Box, Button, Select, Text} from 'native-base';
+import {useCallback, useMemo, useState} from 'react';
+import {useTranslation} from 'react-i18next';
+import {SoilIdProjectSoilSettingsDepthIntervalPresetChoices} from 'terraso-client-shared/graphqlSchema/graphql';
+import {
+  LabelledDepthInterval,
+  ProjectSoilSettings,
+  updateProjectDepthInterval,
+  updateProjectSoilSettings,
+} from 'terraso-client-shared/soilId/soilIdSlice';
+import {ConfirmModal} from 'terraso-mobile-client/components/ConfirmModal';
+import {useProjectRoleContext} from 'terraso-mobile-client/context/ProjectRoleContext';
+import {useDispatch} from 'terraso-mobile-client/store';
+import {DepthIntervalTable} from 'terraso-mobile-client/screens/ProjectInputScreen/DepthIntervalTable';
+import {AddIntervalModal} from 'terraso-mobile-client/components/AddIntervalModal';
+import {DepthPresets} from 'terraso-mobile-client/constants';
+import {Modal} from 'terraso-mobile-client/components/Modal';
+import {Icon} from 'terraso-mobile-client/components/Icons';
+
+export const SoilPitSettings = ({
+  data: settings,
+  projectId,
+}: {
+  data: ProjectSoilSettings;
+  projectId: string;
+}) => {
+  const {t} = useTranslation();
+  const [selectedPreset, setSelectedPreset] =
+    useState<SoilIdProjectSoilSettingsDepthIntervalPresetChoices>(
+      settings.depthIntervalPreset,
+    );
+  const dispatch = useDispatch();
+  const existingIntervals = useMemo(
+    () => settings.depthIntervals.map(interval => interval.depthInterval),
+    [settings.depthIntervals],
+  );
+
+  const projectRole = useProjectRoleContext();
+
+  const userCanUpdateIntervals = useMemo(
+    () => projectRole === 'manager',
```

Comment:
issue: manager should come from a constant.

Reply:
Because manager is part of the Typescript union type, we've generally not been using constants or enums for these kind of things in the code. Typescript can make sure that the string matches the expected type, so creating a constant `MANAGER` is not very useful.